data/campaigns/Eastern_Invasion/scenarios/02_The_Escape_Tunnel.cfg

```diff
@@ -355,8 +395,12 @@
 
         [remove_shroud]
             side=1
-            x,y=2-4,3
+            x,y=2-4,2-3
         [/remove_shroud]
+        [lift_fog]
+            side=1
+            x,y=2-4,2-3
+        [/lift_fog]
```

Comment:
(Not about this line, but github wouldn't let me add the comment where I want it)

Later in this event Dacyn says "They follow us... we must get out of this tunnel. It looks like it is leading east. That is unfortunate, for it will take us beyond the borders of Wesnoth. But it is our only option", but part of the in-world rationale for the changes is that Dacyn _knows_ where the tunnel leads, and he does say earlier "these caves are currently inhabited by trolls" before the party sights the trolls, so shouldn't he be more confident as to where the tunnel's exit will be?

Reply:
I have no idea, I wasn't intending to do anything about dialogue with this PR, besides a couple minor things regarding the shrouds. I actually don't remember where the tunnel exit even is, so someone might need to help me a bit with that if we want to change it. :p

Reply:
Merging this for the string freeze. Could you open an issue and explain / copy the issue there?

Reply:
https://github.com/wesnoth/wesnoth/issues/3848